### PR TITLE
Add .mkv support

### PR DESCRIPTION
**Pull request by qubist**
* Changes video_info function to return an dictionary.
* Get framerate and frame count in a more robust way from the ffmpeg 
probe. This method works for the mkv I was testing on.
* The dictionary returned from video_info now includes duration, setting 
up for future runtime estimates.

Hopefully equivalent to f450a90, baa0526, and 8f0c58a

### CODE REVIEW

File: slowmovie.py [hunk 1]
```diff
@@ -50,11 +50,11 @@ def generate_frame(in_filename, out_filename, time):
         .run(capture_stdout=True, capture_stderr=True)
     )
 
-def check_mp4(value):
+def check_vid(value):
     if not os.path.isfile(value):
         raise configargparse.ArgumentTypeError("File '%s' does not exist" % value)
     if not supported_filetype(value):
-        raise configargparse.ArgumentTypeError("'%s' is not a supported file type" % value)
+        raise argparse.ArgumentTypeError(f"File '{file}' should be a file with one of the following supported extensions: {', '.join(fileTypes)}")
```
Comment (robweber) [inline slowmovie.py:57]: `configargparse.ArgumentTypeError` and `argparse.ArgumentTypeError` are basically the same error by design. We're using one in line 55 and a different one in 57. Should we pick one to be consistent?

**Review by robweber — COMMENTED** [review]
Other than that one comment looking good.

File: slowmovie.py (hunk 1 above)
Comment (qubist) [reply slowmovie.py]: Oops, yes! Copy-paste error. Let's stick with configparse as that's what's imported. Fixed in bb4db81

**Review by missionfloyd — APPROVED** [review]
Looks good

**Pull request merged**